tests/sync/sync_test_utils.cpp

```diff
@@ -56,13 +56,10 @@ bool results_contains_original_name(SyncFileActionMetadataResults& results, cons
 }
 
 std::string tmp_dir() {
-    const char* dir = getenv("TMPDIR");
```

Comment:
I originally thought not using $TMPDIR would make things easier to debug, but upon further thought I don't think that's true. I'll change it back.